src/components/actionSheet/index.tsx

```diff
@@ -101,6 +109,10 @@ type ActionSheetProps = {
    * testID for e2e tests
    */
   testID?: string;
+  /**
+   * Whether to use ScrollView as the container
+   */
+  scrollContainer?: boolean;
```

Comment:
Naming convention in this case is more like 'useScrollView' or 'useScrollContainer' since it's a boolean and not element.
But anyways I wonder if we should add this prop at all or figure out in the component when the children height is greater then the container's and use the scrollview instead of a view.